Doc/tutorial/controlflow.rst

```diff
@@ -519,6 +519,169 @@ and of course it would print:
 Note that the order in which the keyword arguments are printed is guaranteed
 to match the order in which they were provided in the function call.
 
+Special parameters
+------------------
+
+By default, arguments may be passed to a Python function either by position
+or explicitly by keyword. For readability and performance, it makes sense to
+restrict the way arguments can be passed so that a developer need only look
+at the function definition to determine if items are passed by position, by
+position or keyword, or by keyword.
+
+A function definition may look like::
+
+   def f(pos1, pos2, /, pos_or_kwd, *, kwd1, kwd2):
+         -----------    ----------     ----------
+           |             |                  |
+           |        Positional or keyword   |
+           |                                - Keyword only
+            -- Positional only
+
+where ``/`` and ``*`` are optional. If used, these symbols indicate the kind of
+parameter by how the arguments may be passed to the function:
+positional-only, positional-or-keyword, and keyword-only. Keyword parameters
+are also referred to as named parameters.
+
+-------------------------------
+Positional-or-Keyword Arguments
+-------------------------------
+
+If ``/`` and ``*`` are not present in the function definition, arguments may
+be passed to a function by position or by keyword.
+
+--------------------------
+Positional-Only Parameters
+--------------------------
+
+Looking at this in a bit more detail, it is possible to mark certain parameters
+as *positional-only*. If *positional-only*, the parameters' order matters, and
```

Comment:
Not sure "the parameters' order matters" adds anything. It is more than when calling by keyword order does not matter. on pos or keyword order can also matter if called via positional.

Reply:
This is for the case when we use the `range()` parameter order in the PEP. This part is also from the `How to teach this` from the PEP itself.

>It is more than when calling by keyword order does not matter. on pos or keyword order can also matter if called via positional.

I do not understand what you mean :S

Reply:
I mean that if a function is declared with pos or keyword, the order matters in the same way as for postional only if called without keywords. Isn’t it?

Reply:
Yep, but here we are saying that if you use positional-only there is no way of calling with an altered order.